examples/backends/sglang/launch/xpu/disagg_pd_xpu.sh

```
@@ -0,0 +1,523 @@
+        return 1
+    fi
+
+    local test_prompt="The capital of France is"
+    local max_tokens=32
+
+    log_info "Test prompt: \"$test_prompt\""
+    log_info "Max tokens: $max_tokens"
+    echo
+
+    # Run curl test
+    local response=$(curl -s "http://127.0.0.1:$ROUTER_PORT/v1/completions" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)